Public/Src/Cache/ContentStore/Distributed/FileSystem/IFileCopier.cs

```diff
@@ -60,17 +60,19 @@ public interface ITraceableAbsolutePathFileCopier : IAbsolutePathFileCopier, ITr
     }
 
     /// <summary>
-    /// Requests another machine to copy from the current machine.
+    /// Copies files to another machine.
     /// </summary>
-    public interface ICopyRequester
+    public interface ICopySender

```

Comment:
It seems that this interface supports both, the push and the pull models of copying files to the machine.

So, maybe the name should be reflect that...
Like IProactiveCopier? (this one is bad as well, but at least it is vague enough)... Happy to brainstorm the name:)